while-job-hunting.md

```diff
@@ -0,0 +1,17 @@
+# While job hunting
+
+### You're in a strong position
+
+Software developers are in high demand, and the skillset that CodeClan arms you with combined with your previous life experience makes you a great hire for any tech firm.
+
+Don't undersell yourself and don't go for jobs that wouldn't value your skills. Your skillset is worth more than an £18k FTE salary.
```

Comment:
I'd add in an explanation of the acronym `FTE (Full Time Equivalent)` - or just remove it entirely? Not sure it's needed in this context 🤔 

wdyt?